adap/htdocs/register.php

```diff
@@ -0,0 +1,134 @@
+<?php
+  include 'homepage/navbar_before_login.php';
+?>
+<!DOCTYPE html>
+<head>
+  <title>ADAP</title>
+  <!--Import Google Icon Font-->
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+  <!--Import materialize.css-->
+  <link type="text/css" rel="stylesheet" href="css/materialize.min.css"  media="screen,projection"/>
+  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+  <style>
+  span.reg_err {
+    color: red;
+  }
+  </style>
+</head>
+<body>
+  <?php
+    // Connect to the database. Please change the password in the following line accordingly
+    require_once 'config.php';
+    if (isset($_POST['submit'])) { // Submit the update SQL command
+      $company_name = $_POST['company_name'];
+      $company_name_bool = false;
+      $email = $_POST['email'];
+      $email_bool = false;
+      $password = $_POST['password'];
+      $password_bool = false;
+      $contact = $_POST['contact'];
+      $contact_bool = false;
+      $address = $_POST['address'];
+      $address_bool = false;
+      $postal_code = $_POST['postal_code'];
+      $postal_code_bool = false;
+
+      if ($company_name==null) $company_name_err = 'Company name is empty!';
+      else $company_name_bool = true;
+
+      if ($email==null) $email_err= 'Email is empty!';
+      else if (!filter_var($email, FILTER_VALIDATE_EMAIL)) $email_err = 'Email is invalid!';
+      else $email_bool = true;
+      
+      if ($password==null) $password_err = 'Password is empty!';
+      else if (($_POST['password_reenter']==null)||($password<>$_POST['password_reenter'])) $password_comp_err = 'Passwords do not match!';
+      else $password_bool = true;
+
+      if ($contact==null) $contact_err ='Contact Number is empty!';
+      else if (!is_numeric($contact)||(strlen($contact)<3)) $contact_err ='Contact Number is Invalid';
+      else $contact_bool = true;
+
+      if ($address==null) $address_err ='Address is empty!';
+      else $address_bool = true;
+      
+      if ($postal_code==null) $postal_code_err ='Postal code is empty!';
+      else if (!is_numeric($postal_code)||!(strlen($postal_code)==6)) $postal_code_err ='Postal Code is Invalid!';
+      else $postal_code_bool = true;
+
+      if (($company_name_bool == true) && ($email_bool == true) && ($password_bool == true) && ($contact_bool == true) && ($address_bool == true) && ($postal_code_bool == true)) {
+        $result = pg_query($db, "INSERT INTO company_account VALUES ('$company_name','$email','$password','$contact','$address','$postal_code')");
+        if (!$result)  $err = 'Account was already created before. Would you like to <a href="../adap/login.php">log in</a> instead?';
+        else {
+            header("location: register_success.php");
+        }
+      }}
+    ?>
```

Comment:
This definitely works as well. Even though I am not sure if you want to be more consistent with the other pages and shift this to another `php` file under the `php` folder (but that also means `javascript` will come into play). Since time is tight we can forget this for now I guess